config/sitemap.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+# Set the host name for URL creation
+SitemapGenerator::Sitemap.default_host = ActionDispatch::Http::URL.url_for(Rails.application.routes.default_url_options)
+
+SitemapGenerator::Sitemap.create do
+  # Put links creation logic here.
+  #
+  # The root path "/" and sitemap index file are added automatically for you.
+  # Links are added to the Sitemap in the order they are specified.
+  #
+  # Usage: add(path, options={})
+  #        (default options are used if you don"t specify)
+  #
+  # Defaults: :priority => 0.5, :changefreq => "weekly",
+  #           lastmod: Time.now, :host => default_host
+  #
+  # Examples:
+  #
+  # Add "/articles"
+  #
+  #   add articles_path, :priority => 0.7, :changefreq => "daily"
+  #
+  # Add all articles:
+  #
+  #   Article.find_each do |article|
+  #     add article_path(article), lastmod: article.updated_at
+  #   end
+
+  add "/about"
+  add "/visit-study"
+  add "/explore-charles"
+  add "/research-services"
+  add "/blockson"
+  add "/contact"
+  add "/home"
```

Comment:
We don't have anything at this URL? It just redirects to `/`, right? So I think this should this just be `/`